acceptance/bundle/migrate/basic/output.txt

```diff
@@ -0,0 +1,99 @@
+
+>>> musterr [CLI] bundle deployment migrate
+This command migrates existing terraform state file (terraform.tfstate) to direct deployment engine state file (resources.json). However, no existing local or remote state found.
+
+To start using direct engine, deploy with DATABRICKS_BUNDLE_ENGINE=direct env var or bundle.engine="direct" in databricks.yml.
+
+>>> DATABRICKS_BUNDLE_ENGINE=terraform [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/migrate-basic-test/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle deployment migrate
+Migrated 3 resources to direct engine state file: [TEST_TMP_DIR]/.databricks/bundle/default/resources.json
+To finalize deployment, run "bundle deploy".
+
+>>> ls -1 .databricks/bundle/default/
+deployment.json
+resources.json
+sync-snapshots
+terraform
+
+>>> jq .state."resources.pipelines.test_pipeline".state.tags.volume_storage_location out.new_state.json
+"s3://deco-uc-prod-isolated-aws-us-east-1/metastore/[UUID]/volumes/[UUID]"
+
+>>> DATABRICKS_BUNDLE_ENGINE=terraform musterr [CLI] bundle plan
+Error: Required engine "terraform" does not match present state files. Set required engine via "DATABRICKS_BUNDLE_ENGINE" env var.
```

Comment:
... or leave it empty to infer the right engine.

Reply:
updated